addons/mail/models/mail_link_preview.py

```diff
@@ -89,37 +89,44 @@ def _is_domain_throttled(self, url):
     @api.model
     def _get_link_preview_from_url(self, url, request_session):
```

Comment:
can we make `request_session` optionnal and fallback to `requests.Session()` if None
```suggestion
    def _get_link_preview_from_url(self, url, request_session=None):
```

so it will simplify the caller in the next commit